static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -160,7 +161,25 @@ export function DraggableTabBar({
         organization,
       });
     }
-  };
+  }, [onSave, organization, setTabs, tabListState?.selectedKey, tabs]);
+
+  useEffect(() => {
```

Comment:
The `useHotkeys` hook might come in handy here.

Reply:
did not know about that but that looks way cleaner, thanks! will update it now